nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -332,11 +334,20 @@ define([
                 initialParamStates[fileType] = 'incomplete';
                 [fileParamIds[fileType], otherParamIds[fileType], outputParamIds[fileType]] =
                     filterFileParameters(spec);
+                // if there's just a file and an output
```

Comment:
Short version - if the app spec only has a single input, that means we're populating and suggesting an output name. So make sure it's sanitized to be a legal workspace object name, and slap the suffix on there if we have one.